src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -84,6 +84,30 @@ public function dump($input, $inline = 0, $indent = 0, $flags = 0)
             $isAHash = array_keys($input) !== range(0, count($input) - 1);
 
             foreach ($input as $key => $value) {
+                if ($inline - 1 > 0 && Yaml::DUMP_MULTI_LINE_AS_BLOCK & $flags && is_string($value) && false !== strpos($value, "\n")) {
```

Comment:
I would say ``$inline > 1``, to avoid an operation (even though OPCache might be able to detect this optimization)